CHANGELOG.md

```diff
@@ -2,12 +2,42 @@
 
 ## Unreleased
 
+## 3.13.0 (Feature release)
+
+### New features
+
+#### Add a 'none' option and 'or' divider to checkboxes
+
+You can now add a 'none' option to checkboxes. For example, 'None of the above' or 'None of these options apply to me.'
+
+Use the 'none' option to [allow users to state that none of the checkboxes apply to them](https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none-). Otherwise, users would have to work out that they should leave all the checkboxes unticked.
+
+You can now also add an 'or' divider before the 'none' option. Use the 'or' divider to make the 'none' option stand out from the other options so it's more visible to users.
```

Comment:
You could probably simplify. For example: 

- "Use the 'or' divider to make the 'none' option more visible to users"
- "Use the 'or' divider to make the 'none' option stand out from the other options"

Reply:
```suggestion
You can now also add an 'or' divider before the 'none' option. Use the 'or' divider to make the 'none' option more visible to users.
```